### PR TITLE
OADP: Set Up Periodic Jobs and  Slack Alerts for  Results

### PR DESCRIPTION
**Pull request by hhpatel14**
This change creates periodic jobs at 4 am UTC(jobs run on UTC time zone so 4am UTC = 12am EST). Also, the change reports failure to given slack channel with specific job name if there is any.

### CODE REVIEW

**Comment by alaypatel07** [comment]
this needs some re-work from what I am understanding in the docs

/hold
/wip

**Comment by alaypatel07** [comment]
/lgtm

I will keep it on hold for Shawn or Dylan to take a final look

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml [hunk 1]
```diff
@@ -57,6 +57,30 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: optional-operators-ci-aws
+- as: operator-e2e-periodic-slack
+  cron: 0 4 * * *
```
Comment (shawn-hurley) [inline ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml:61]: Sorry, I might be wrong on this, but is this running every hour on the 4th min? Is that what we want that seems like a lot of runs?

**Review by shawn-hurley — CHANGES_REQUESTED** [review]
want to make sure on the period of the run

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml (hunk 1 above)
Comment (hhpatel14) [reply ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml:61]: This runs at 4am UTC = 12am EST.  "*/4 * * * *" runs at every 4th minute, and "0 */4 * * *" runs at every 4th hour.

Comment (shawn-hurley) [reply ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml:61]: Thanks, had to go do a refresh on cron syntax!

**Comment by shawn-hurley** [comment]
/lgtm

@dymurray want to un hold it?

**Review by dymurray — APPROVED** [review]

**Pull request merged**